var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -427,6 +427,18 @@ def ensure_parallel_compiler_wrappers(self):
                 # No try/except here, fix the condition above instead:
                 symlink('h5fc', 'h5pfc')
 
+    @run_after('install')
+    def symlink_to_h5hl_wrappers(self):
+        # CMake's FindHDF5 relies only on h5cc so it doesn't find
```

Comment:
CMake's FindHDF5 was designed to find the autotools built library.
CMake has find_package which allows more flexibility.

Reply:
In my opinion it seems intractable to have all the projects that use `FindHDF5` use `find_package` instead.